src/pages/colinks/give/ShareGiveCard.tsx

```diff
@@ -0,0 +1,314 @@
+import React, { useState } from 'react';
+
+import copy from 'copy-to-clipboard';
+import { skillTextStyle } from 'stitches.config';
+
+import { Button, Flex, Link, Text, HR } from '../../../ui';
+import { DrawerModal } from '../../../ui/DrawerModal';
+import { useToast } from 'hooks';
+import {
+  Copy,
+  FarcasterActions,
+  GemCoOutline,
+  MessageHeartOutline,
+  ShareGive,
+  Wand,
+} from 'icons/__generated';
+import { PartyDisplayText } from 'ui/Tooltip/PartyDisplayText';
+
+import { codeStyle } from './GiveBotCard';
+import { LearnCard, contentStyles } from './LearnCard';
+
+export const ShareGiveCard = ({ skill }: { skill?: string }) => {
+  const [modalVisible, setModalVisible] = useState(false);
+  return (
+    <>
+      <LearnCard
+        image="/imgs/background/give-share.jpg"
+        onClick={() => setModalVisible(true)}
+      >
+        <Flex
+          column
+          css={{
+            ...contentStyles,
+            justifyContent: 'space-between',
+            background:
+              'radial-gradient(circle at 25% 0%, rgb(172 116 20 / 50%) 20%, rgba(32, 5, 64, 0.11) 100%)',
+          }}
+        >
+          <Flex
+            column
+            css={{
+              alignItems: 'center',
+              gap: '$sm',
+              justifyContent: 'center',
+              flexGrow: 1,
+            }}
+          >
+            <ShareGive size="2xl" fa />
+            <Flex
+              css={{
+                columnGap: '$sm',
+                flexWrap: 'wrap',
+                textAlign: 'center',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+            >
+              <Text
+                size="large"
+                semibold
+                css={{
+                  whiteSpace: 'nowrap',
+                }}
+              >
+                Share{' '}
+              </Text>
+              <Text
+                size="large"
+                semibold
+                css={{
+                  ...skillTextStyle,
+                  maxWidth: '16rem',
+                  textTransform: 'capitalize',
+                }}
+              >
+                {skill}
+              </Text>
+            </Flex>
+          </Flex>
+        </Flex>
+      </LearnCard>
+      {modalVisible && (
+        <DrawerModal
+          visible={modalVisible}
+          closeButtonStyles={{ color: '$white80' }}
+          onClose={() => setModalVisible(false)}
+        >
+          <ShareGiveContent skill={skill} />
+        </DrawerModal>
+      )}
+    </>
+  );
+};
+
+export const ShareGiveContent = ({ skill }: { skill?: string }) => {
+  const startPartyUrl = `https://warpcast.com/~/compose?text=https://give.party/${skill}&embeds[]=https://give.party/${skill}`;
```

Comment:
i wonder if this works with this skill: https://coordinape.com/give/skill/%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA%20%F0%92%90%AA

haha

Reply:
𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪𒐪